src/v2/Apps/Partner/Routes/Articles/index.tsx

```diff
@@ -40,12 +46,15 @@ const Articles: React.FC<ArticlesProps> = ({ partner, relay }) => {
         }
       }
     )
+
+    router.push({
+      pathname: location.pathname,
+      query: { ...location.query, page },
+    })
```

Comment:
It's not immediately clear to me from this PR but we need: the pagination to have actual hrefs so that they are crawlable; and the page param has to be supported in the routes as well.